libraries/ui/src/Footer.tsx

```diff
@@ -11,73 +11,105 @@ export type FooterProps = React.PropsWithChildren<{
   logo?: string
 }>;
 
+type FooterSectionProps = {
+  title?: string;
+  links?: { href: string; label: string }[];
+};
+
+const FooterLinksSection: React.FC<FooterSectionProps> = ({ title, links }) => (
+  <div className="footer__section">
+    {title && <p className="footer__heading font-[650] text-on-dark mb-4">{title}</p>}
+    {links && (
+      <ul className="footer__list space-y-2 mb-auto font-normal list-none p-0">
+        {links.map((link) => (
+          <li key={link.href} className="footer__item">
+            <a href={link.href} className="footer__link text-bluedot-lighter hover:text-white hover:cursor-pointer">
+              {link.label}
+            </a>
```

Comment:
I decided to keep these font sizes at the default 14px rather than the 16px in the designs